README.md

```diff
@@ -1,2 +1,350 @@
+    ],
+    "authentication": [
+      "did:itn:NL28Bubaa8xUmhJAARNkNS#z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC"
+    ],
+    "id": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+    "keyAgreement": [
+      {
+        "id": "did:itn:NL28Bubaa8xUmhJAARNkNS#z6LSd76Tk2UJ3RVxcTjYjZ8TfvHMQgZG5HmbKPmCidPXyTGZ",
+        "type": "X25519KeyAgreementKey2020",
+        "controller": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+        "publicKeyMultibase": "z6LSd76Tk2UJ3RVxcTjYjZ8TfvHMQgZG5HmbKPmCidPXyTGZ"
+      }
+    ],
+    "verificationMethod": [
+      {
+        "id": "did:itn:NL28Bubaa8xUmhJAARNkNS#z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC",
+        "type": "Ed25519VerificationKey2020",
+        "controller": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+        "publicKeyMultibase": "z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC"
+      }
+    ]
+  },
+  "didDocumentMetadata": {
+    "created": "2023-11-10T15:20:59Z",
+    "updated": "2023-11-10T15:20:59Z",
+    "deactivated": false,
+    "recoveryCommitment": "99cdefca761b58664641d3f0b89b4f1e04ad833b7a24a726e3424bc6c17f39a0"
+  },
+  "didResolutionMetadata": { }
+}
+```
+
+## Security Considerations
+
+For all `did:itn` DIDs, the initial asset creation, and subsequent updates are executed using `Ed25519` keys, which are widely recognized as a robust and secure cryptographic mechanism. `Confirm With Umed & Andreas - if the keys are different. DELETE THIS COMMENT LATER.`
+
+- Authentication service utilizing [OAuth2](https://oauth.net/2/) for REST APIs and [DIF DID AuthN](https://identity.foundation/working-groups/authentication.html) for DIF DIDcomm messaging
+- Keys are specified in the DID Document in Verification Methods object for various Verification Relationships.
+- ITN heavily relies on W3C DIDs (Decentralized Identifiers) and VCs (Verifiable Credentials) employ cryptographic mechanisms and decentralized trust models to mitigate the risks associated with various forms of attacks.
+- Within ITN, actors of [W3C VC Data Model](https://www.w3.org/TR/vc-data-model-2.0/) - Issuer, Holder, and Verifier MUST be an ITN DID associated with corresponding DID Document.   
+
+Here's how ITN can prevent the specific attacks outlined:
+
+1. Eavesdropping
+
+ITN DID can facilitate the establishment of secure communication channels between parties by encrypting messages exchanged between DIDs using cryptographic keys defined in the corresponding DID Documents. DIDs and VCs use cryptographic signatures to ensure that communication between parties is encrypted and authenticated. This prevents unauthorized interception and eavesdropping of sensitive information.
+
+2. Replay
+
+VCs typically include timestamps & cryptogrphic signtaures to prevent replay attacks. Cryptographic signatures are generated using the cryptographic keys are defined in the corresponding DID Documents. Additionally, ITN DID can be combined with secure channels (e.g., HTTPS) to prevent unauthorized retransmission of captured communication.
```

Comment:
```suggestion
VCs typically include timestamps & cryptographic signatures with nonces to prevent replay attacks using the cryptographic keys defined in the corresponding DID Documents.
```